### DISCUSSION TITLE
creating and mofifying pivots using ImportExcel

### DISCUSSION THREAD

**Discussion by jp-villette** [dfinke/ImportExcel#1411] [General]
Hello,

I am using ImportExcel to create a basic report using pivots .

When created, i tried to update the different titles in each pivot. I am foccussing here on three of them, in the corner left of the pivots. they have by default the values: 
- Count of .... the title of the original column used to build this pivot,
- Column Labels and Row labels.

I am trying to change the "column labels" and the "Row labels. Changing the corresponding cells does not work, it comes back to the default values when the excel file is open. Changing the caption values in the PivotTables.RowFiels or PivotTables.ColumnFiels definitions does not works ever, it seems to be read only.

Basically, how can i change these two values ?

Thanks in advance for your help.
Regards,
JPV

**Comment by jp-villette** [comment]
Got it, the re-setting of the two captions  (RowHeaderCaption and ColumnHeaderCaption) at the PivotTables level failed because of protected sheets.

JPV
